dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -363,6 +364,61 @@ public Collection updateCollection(@ApiParam(hidden = true) @Auth User user,
 
     }
 
+    @PUT
+    @Timed
+    @Path("{organizationId}/collections/{collectionId}/description")
+    @UnitOfWork
+    @ApiOperation(value = "Update an collection's description.", notes = "Description in markdown", authorizations = {
+            @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Collection.class)
+    public Collection updateCollectionDescription(@ApiParam(hidden = true) @Auth User user,
+            @ApiParam(value = "Collection to update description.", required = true) @PathParam("organizationId") Long organizationId,
```

Comment:
Value should mention organization id, not collection.

Also, we should be consistent on whether these end with periods or not (the one on line 376 doesn't) -- I don't know what the most common pattern is.